doc/lua_api.md

```diff
@@ -9647,6 +9652,10 @@ in active mapblocks.
 
 ```lua
 {
+    name = "".
+    -- Optional name of ABM.
+    -- ABM with name can be changed at runtime (See `core.override_abm`).
```

Comment:
It might be better to document the changeable fields in `core.override_abm` instead. That reduces duplications, and moves the information to the function, where it is effectively used.

Reply:
I do not think it is a good idea to have part of the supported fields documented somewhere in a different place.